docs/DeveloperGuide.md

```diff
@@ -2,8 +2,34 @@
 layout: page
 title: Developer Guide
 ---
-* Table of Contents
-{:toc}
```

Comment:
I think this line will generate a table of content, so we don't have to manually write one which is prone to error.

Reply:
Thanks